Tilers/CityTiler/CityTilerDBConfigReference.yml

```diff
@@ -1,5 +0,0 @@
-PG_HOST: <FQDN or IP number of the server hosting your db>
```

Comment:
CityTilerDBConfig.yml is not be commited. But CityTilerDBConfigReference.yml is a reference file that ones copies to CityTilerDBConfig.yml and customises...
Please restore this file.